tesseract_motion_planners/trajopt/src/trajopt_motion_planner.cpp

```diff
@@ -412,15 +412,27 @@ TrajOptMotionPlanner::createProblem(const PlannerRequest& request) const
       const auto& seed_composite = seed_flat_pattern[i].get().as<tesseract_planning::CompositeInstruction>();
       auto interpolate_cnt = static_cast<int>(seed_composite.size());
 
+      // Get Plan Profile
       std::string profile = getProfileString(name_, plan_instruction.getProfile(), request.plan_profile_remapping);
       TrajOptPlanProfile::ConstPtr cur_plan_profile = getProfile<TrajOptPlanProfile>(
           name_, profile, *request.profiles, std::make_shared<TrajOptDefaultPlanProfile>());
       cur_plan_profile = applyProfileOverrides(name_, profile, cur_plan_profile, plan_instruction.profile_overrides);
       if (!cur_plan_profile)
         throw std::runtime_error("TrajOptPlannerUniversalConfig: Invalid profile");
 
+      // Get Plan Path Profile: Default is an empty string
+      std::string path_profile = plan_instruction.getPathProfile();
+      path_profile = getProfileString(name_, path_profile, request.plan_profile_remapping, "");
```

Comment:
Should we have a path profile override dictionary as well? Otherwise I think the names could clash right?

Reply:
I don't think so because they are both accessing plan profiles at the dictionary because we are not making a distinction between plan and path profiles at the moment.